src/encoded/static/components/matrix_human_donor.js

```diff
@@ -0,0 +1,744 @@
+            }
+        });
+        row.diseases = [...new Set(row.diseases)];
+        y.push(row);
+    });
+
+    return { x, y };
+};
+
+/**
+ * Populate the matrix
+ *
+ * @param {object} dataTableData Matrix data
+ * @param {string} baseUrl Url of the page from context
+ * @param {string} searchBaseUrl Search url from context
+ * @param {object} graph based on context
+ */
+const convertToMatrixData = (dataTableData, baseUrl, searchBaseUrl, graph) => {
+    const { x, y } = dataTableData;
+    const matrixData = [];
+    const rowColor = '#fff';
+    const firstColumnWidth = '400px'; // changes the default width  in grid matrix
+
+    // header
+    const header = {
+        type: 'matrixHeader',
+        itemKey: 'matrix-header-key',
+        items: [{
+            content: (
+                <div style={{ display: 'flex', flexDirection: 'row', width: firstColumnWidth }} key="matrix-header-key-accession">
+                    <div style={{ border: '1px solid #f0f0f0', flex: '0 1 32%', textDecoration: 'none', whiteSpace: 'nowrap', textAlign: 'center' }}>
+                        <div className="grid-matrix__header__text">
+                                Accession
+                        </div>
+                    </div>
+                    <div style={{ border: '1px solid #f0f0f0', flex: '0 1 40%' }} key="matrix-header-key-age">
+                        <div className="grid-matrix__header__text">
+                                Age
+                        </div>
+                    </div>
+                    <div style={{ border: '1px solid #f0f0f0', flex: '0 1 8%' }} key="matrix-header-gender">
+                        <div className="grid-matrix__header__text">
+                                Sex
+                        </div>
+                    </div>
+                    <div style={{ border: '1px solid #f0f0f0', flex: '0 1 20%' }} key="matrix-header-key-disease">
+                        <div className="grid-matrix__header__text">
+                                Disease
+                        </div>
+                    </div>
+                </div>
+            ),
+            itemKey: 'matrix-header-key-titles',
+        }].concat(dataTableData.x.header.map((xData) => ({
+            content: <a href={`${baseUrl.replace('/human-donor-matrix/?type', '/search/?type')}&assay_title=${xData}`}>{xData}</a>,
+            itemKey: xData.replace(/\s+/g, ''),
+        }))),
+    };
+
+    matrixData.push(header);
+
+    y.forEach((yEntry) => {
+        const accessionTextColor = isLight(rowColor) ? '#000' : '#fff';
+        const { accession, assayTotals } = yEntry;
+        const maxAssayTotal = Math.max(...assayTotals);
+        const minAssayTotal = Math.min(...assayTotals);
+        const logBase = Math.log(1 + maxAssayTotal + minAssayTotal);
+
+        // change grid matrix default values
+        const rowHeaderFirstColStyle = {
+            color: `${accessionTextColor}`,
+            backgroundColor: `${rowColor}`,
```

Comment:
Don’t need the string templates here since you don’t add anything to these variables, so you can just use the variables alone.